rotkehlchen/tests/unit/decoders/test_superchain_bridge.py

```diff
@@ -457,46 +489,164 @@ def test_withdraw_dai_on_ethereum(ethereum_inquirer, ethereum_accounts):
             event_type=HistoryEventType.SPEND,
             event_subtype=HistoryEventSubType.FEE,
             asset=A_ETH,
-            balance=Balance(amount=FVal(gas_amount)),
+            balance=Balance(amount=FVal('0.002095833929196665')),
```

Comment:
As in other tests move the `'0.002095833929196665'` as a variable above with the rest. Same for the bridging amount. Saves repeat typing. Do that for all your other tests too